qa/tasks/cephfs/test_failover.py

```diff
@@ -706,6 +706,66 @@ def test_rank_stopped(self):
         status = self._confirm_single_replay()
         self.assertTrue(standby_count, len(list(status.get_standbys())))
 
+    def is_there_only_1_cache_oversized_warn(self):
+        health_report = self.get_ceph_cmd_stdout('health detail')
+        log.info(health_report)
+        for line in health_report.split('\n'):
+            if line.find('MDS_CACHE_OVERSIZED') != -1:
+                self.assertIn('1 MDSs report oversized cache', line)
+                return True
+        return False
+
+    def test_health_warn_oversize_cache(self):
+        '''
+        Test that when MDS cache size crosses the limit, health warning is
+        not printed standy-replay MDS.
+
+        Tests: https://tracker.ceph.com/issues/63514
+        '''
+        # reduce MDS cache limit, default MDS cache limit is too high which
+        # will unnecessarily consume too many resources and too much time.
+        self.run_ceph_cmd('config set mds mds_cache_memory_limit 10M')
+        # health warning for crossing MDS cache size limit won't be raised
+        # until a threshold. default threshold is too high. it will
+        # unnecssarily consume so much time and resources.
+        self.run_ceph_cmd('config set mds mds_health_cache_threshold 1.000001')
+        # so that there is only active MDS and only 1 health warning is
+        # produced. presence of 2 warning should cause this test to fail
+        self.fs.set_max_mds(1)
+        self.run_ceph_cmd(f'fs set {self.fs.name} allow_standby_replay true')
+        # although ideally not required, leaving some time for cluster to
+        # bring above configs in effect.
+        time.sleep(5)
```

Comment:
You choose be waiting till max_mds equals 1 and there is a s-r daemons. There are helpers for this.